index.js

```diff
@@ -556,6 +556,12 @@ prototype.buildParameter = function(oldParameter) {
     required: fixNonStringValue(oldParameter.required)
   });
 
+  Object.keys(oldParameter).forEach(function(property) {
```

Comment:
It should be according to Swagger 1.2 spec.
But it's better to use `this.forEach` construct because it already have [check](https://github.com/apigee-127/swagger-converter/blob/master/index.js#L768) in it.